handlingData.rb

```diff
@@ -0,0 +1,146 @@
+module Check_applicability
+
+  def loan_applicability?(population, gdp, states, state)
+    return true if population > 200000 && gdp >= 20000 && states >= 5 && state == "developing"
+    false
+  end
+
+  def seat_applicability?(population, gdp, states, state)
+    return true if population > 200000 && gdp >= 10000 && states >= 8 && state == "developed"
+    false
+  end
+
+  def win_war?(population, states, army_strength)
+    return true if population > 50000 && states >= 10 && army_strength >= 20000
+    false
+  end
+
+end
+
+class Country
+  extend Check_applicability
+
+  attr_accessor :Population, :GDP, :States, :Army_Strength, :State_of_country, :country_name, :country_code
```

Comment:
In ruby, we follow snake_case for naming variable so `Population` => `population` , `Army_Strength` => `army_strength`